song-server/src/main/java/bio/overture/song/server/service/analysis/AnalysisServiceImpl.java

```diff
@@ -208,6 +218,63 @@ public void updateAnalysis(
     return analyses;
   }
 
+  @Override
+  public GetAnalysisResponse getAnalysis(
+      @NonNull String studyId,
+      @NonNull Set<String> analysisStates,
+      @NonNull int page,
+      @NonNull int size) {
+    studyService.checkStudyExist(studyId);
+    resolveSelectedAnalysisStates(analysisStates);
+
+    val sort = new Sort(Sort.Direction.ASC, ANALYSIS_ID);
```

Comment:
What do you think about making the sort a requestParam too? Like have a set for `fieldNames` to sort on and the `order` to chose asc or desc?? If we prefer/require only a default sort, maybe something like `publishedAt` or `createdAt` makes more sense??

Reply:
I checked a bunch of analyses and there are quite a few of them (same program)that are created at the same time, so sorting by `createdAt` doesn't provide much functionality. Not sure about `publishedAt`.

Reply:
Hmm ok, ids are always unique so order_by/sort on them will always return the same result. I suppose if we just want the default sort with no option to sort via params, analysisId (with the studyId we already get) is good enough.